x/airdrop/types/keys.go

```diff
@@ -0,0 +1,26 @@
+package types
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+const (
+	ModuleName = "airdrop"
+	StoreKey = ModuleName
+	RouterKey = ModuleName
+)
+
+var (
+	ActiveFundKeyPrefix = []byte{0x02}
```

Comment:
why `0x02` ?

Reply:
A copy and paste job. Have updated to `0x01`